spec.md

```diff
@@ -975,6 +1173,14 @@ message ValidateVolumeCapabilitiesRequest {
   // Attributes of the volume to check. This field is OPTIONAL and MUST
   // match the attributes of the Volume identified by `volume_id`.
   map<string, string> volume_attributes = 3;
+
+  // Specifies where (regions, zones, racks, etc.) the caller believes
+  // the volume is accessible from.
+  // An caller may specify multiple topologies to indicates they believe
+  // the volume to be accessible from multiple locations.
+  // This field is OPTIONAL. This field MAY only be set if the plugin
```

Comment:
Using MAY in this way doesn't seem correct. Consider: "This field SHALL NOT be set unless the plugin advertises the `ACCESSIBILITY_CONSTRAINTS` capability.`